manifests/g/GPSoftware/DirectoryOpus/12.30/GPSoftware.DirectoryOpus.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: GPSoftware.DirectoryOpus
+PackageVersion: "12.30"
+PackageLocale: en-US
+Publisher: GPSoftware
+# PublisherUrl:
+# PublisherSupportUrl:
+# PrivacyUrl:
+# Author:
+PackageName: Directory Opus
+PackageUrl: https://www.gpsoft.com.au
+License: Copyright (c) GPSoftware
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

Reply:
I just changed the version and hash256 and kept others unchanged. What kind of license should I write here? Should I look up the license 'directory opus' has?